chronograf.go

```diff
@@ -736,20 +736,35 @@ type OrganizationsStore interface {
 	DefaultOrganization(ctx context.Context) (*Organization, error)
 }
 
+// Config is the global application Config for parameters that can be set via
+// API, with different sections, such as Auth
+type Config struct {
+	Auth      AuthConfig        `json:"auth"`
+	LogViewer LogViewerUIConfig `json:"logViewer"`
+}
+
 // AuthConfig is the global application config section for auth parameters
 type AuthConfig struct {
-	// SuperAdminNewUsers should be true by default to give a seamless upgrade to
-	// 1.4.0 for legacy users. It means that all new users will by default receive
-	// SuperAdmin status. If a SuperAdmin wants to change this behavior, they
-	// can toggle it off via the Chronograf UI, in which case newly authenticating
-	// users will simply receive whatever role they would otherwise receive.
 	SuperAdminNewUsers bool `json:"superAdminNewUsers"`
 }
 
-// Config is the global application Config for parameters that can be set via
-// API, with different sections, such as Auth
-type Config struct {
-	Auth AuthConfig `json:"auth"`
+// LogViewerUIConfig is the config sections for log viewer section of the application
+type LogViewerUIConfig struct {
```

Comment:
Why not call  this `LogViewerConfig`?

Reply:
We called it LogViewerUIConfig because it only changes configurations relating to the appearance of the log viewer on the frontend, and not the actual functionality of the log viewer. It's possible that in the future, we would have a separate LogViewerConfig struct relating to log viewer functionality, so the idea was to keep those two concepts separate.

I'm open to renaming it LogViewerConfig, and perhaps having a UI property on the struct.

Reply:
> I'm open to renaming it LogViewerConfig, and perhaps having a UI property on the struct.

Makes sense. I'd probably do something along the lines of what you suggested about 👍